src/Item.java

```diff
@@ -1,8 +1,11 @@
 
 public class Item {
```

Comment:
Implement Comparable here
'
```suggestion
public class Item implements Comparable<Item>{
```